src/core_modules/capture-core/components/D2Form/SectionDescriptionBox.component.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as React from 'react';
```

Comment:
NIT: I would prefer `import React from 'react'` and importing potential react type definitions individually.